packages/shell-api/src/helpers.spec.ts

```diff
@@ -131,7 +131,7 @@ describe('getPrintableShardStatus', function () {
   const testServer = startSharedTestServer();
 
   let mongo: Mongo;
-  let configDatabase: Database;
+  let configDatabase: DatabaseImpl;
```

Comment:
Why not `Database`?

Reply:
These are all fallout from the Database vs DatabaseImpl and Collection vs CollectionImpl thing above. But also as it stands Database & Collection are now generic which means every unit test in the codebase that uses those have to specify the generic types which.. would be a very large PR. And unit tests generally don't care about the schema provided by the generics - they just test the base Collection and Database classes which are just called DatabaseImpl and CollectionImpl now.

I think I can probably "solve" it by renaming CollectionImpl back to Collection and DatabaseImpl back to Database then making either derived classes or types called CollectionWithSchema and DatabaseWithSchema.

Reply:
DatabaseImpl is gone now.